### PR TITLE
Update Falco to v2.11

### PR DESCRIPTION
**Pull request by Molter73**
## Description

Update Falco version. Upstream version base is tag 0.17.2.

Went through the changelog for the last 2 minor versions and the only breaking change is to the filtering mechanism in sinsp, we do a very minor usage of this functionality for getting container events only, we should not be affected by it. All other changes didn't look like they could break anything on our end.

stackrox/falcosecurity-libs#80 fixes any compilation/runtime errors that may be found during testing, it needs to be merged before this PR.

## Checklist
- [x] Investigated and inspected CI test results
- [ ] Updated documentation accordingly

**Automated testing**
  - [ ] Added unit tests
  - [ ] Added integration tests
  - [ ] Added regression tests

If any of these don't apply, please comment below.

## Testing Performed

CI should be enough, but running tests on all archs for good measure.

### CODE REVIEW

File: kernel-modules/MODULE_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.10.0
+2.11.0
```
[the remark this answers is not in the document]
Comment (Molter73) [reply kernel-modules/MODULE_VERSION:1]: You are right, but now that we don't actually need this version, I wonder if we should just change it to something more meaningful, maybe the libs version we are based on? Or even just have it match the collector version or something.

**Review by Stringy — APPROVED** [review]
LGTM!

File: kernel-modules/MODULE_VERSION (hunk 1 above)
[the remark this answers is not in the document]
Comment (Stringy) [reply kernel-modules/MODULE_VERSION]: Or just remove it, I guess. Everything is implicitly tied to the collector version / libs commit now

[the remark this answers is not in the document]
Comment (Molter73) [reply kernel-modules/MODULE_VERSION]: Yeah, I'm thinking more from the point of view of what we print to our logs, but we could also just not print anything.

For now I would keep it and we can do something about this in a follow up PR.

**Pull request merged**